aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -168,13 +168,18 @@ def _accelerate_workflow_actions(
             actions += [install_action, CleanUpAction(self.dependencies_dir)]
             if self.combine_dependencies:
                 # Auto dependency layer disabled, first build
-                actions += [esbuild_with_deps, CopyDependenciesAction(source_dir, scratch_dir, self.dependencies_dir)]
+                actions += [
+                    esbuild_with_deps,
```

Comment:
Does this require an experimental check? Python requires it and node es build does not?

Reply:
This was a conscious decision. Since is esbuild is still in beta, we figured it would be redundant to have two beta flags for it.